pkg/security/api/types.go

```diff
@@ -5,6 +5,11 @@ import (
 	"k8s.io/kubernetes/pkg/api/unversioned"
 )
 
+// +genclient=true
+
+// Security is just a name for the client generated
+type Security struct{}
```

Comment:
what you mean it will always fail? we don't want to generate client for security ?